examples/README.md

```diff
@@ -0,0 +1,87 @@
+# Usage Examples
+
+This file shows how to use the Prospero CLI tool to create an installation of the Wildfly application server. These instructions
+are provided for development and demonstrational purposes only. In practice, no channels are expected to be published for
+Wildfly (upstream project), and the Prospero tool would only be used to provision the JBoss EAP (donwstream product).
+
+Prospero needs two required inputs, to be able to provision an installation:
+
+* a Galleon *feature pack location*,
+* a *Wildfly Channel* reference.
+
+## Dictionary of Terms
+<dl>
+    <dt>Galleon Feature Pack</dt>
+    <dd>
+        The *Galleon feature pack* is a package containing all the metadata required to compose an installation of the Wildfly 
+        or JBoss EAP application server. Among other things, the feature pack contains coordinates (GAVs) to Maven artifacts
+        that the Wildfly server is composed of.
+    </dd>
+    <dt>Wildfly Channel</dt>
+    <dd>
+        The *Wildfly Channel* is another bit of metadata. It's composed of two things: a reference to a
+        *Wildfly Channel manifest*, and a list of Maven repositories where above Maven artifacts can be found.
+    </dd>
+    <dt>Wildfly Channel Manifest</dt>
+    <dd>
+        The *Wildfly Channel Manifest* specifies versions of all the Maven artifacts that given feature
+        pack depends on (these version override the original Maven artifacts versions defined in the feature pack).
+    </dd>
+</dl>
+
+### Wildfly Channels & Manifests
+
+Typically, the Wildfly Channels and Wildfly Channel Manifests would be distributed as well in a form of Maven artifacts and they
+would be referenced via Maven coordinates (GAVs). This directory contains sample channel and manifest files to use when no
+channels / manifests have been officially published yet:
+
+* [wildfly-core-channel.yaml](wildfly-core-channel.yaml),
+* [wildfly-core-manifest.yaml](wildfly-core-manifest.yaml).
+
+(Note: These files have been generated for Wildfly Core version 20.0.0.Beta2.)
+
+These files can be used to provision a Wildfly Core installation in following ways:
+
+## Pre-requisite: Build Prospero
+
+```shell
+cd $PROSPERO_SOURCE_ROOT/
+mvn clean package
+```
+
+## Installation of a Pre-Defined Product Version
+
+```shell
+./prospero install --fpl wildfly \
+  --manifest examples/wildfly-27.0.0.Alpha2-manifest.yaml \
+  --dir installation-dir
+```
+
+(Note: the `--manifest ...` option would be optional for predefined "eap-*" options.)
+
+## Installation Referencing a Channel
+
+```shell
+./prospero install --fpl org.wildfly.core:wildfly-core-galleon-pack:20.0.0.Beta2 \
```

Comment:
If I remove the version, I think the example is going to stop working in the future (when the core-galleon-pack does changes that would not be reflected by the wf-core manifest versioned in this directory).

Maybe I could stress that this readme is intended to developers, not users? Maybe also point out that the FP version would normally be omitted?

BTW my goal was to have some inputs for immediate testing / demonstration, as there are no available manifests around yet and I always have to go and generate a fresh manifest from Wildfly. (I wanted to use wf-core because it's smaller / faster.)

Reply:
I should fix the 20.0.0.Beta3-SNAPSHOT manifest to 20.0.0.Beta2.

Reply:
How does missing `org.jboss.invocation:jboss-invocation-jakarta` manifest? I don't see that being a wildfly-core dependency.

Reply:
> If I remove the version, I think the example is going to stop working in the future (when the core-galleon-pack does changes that would not be reflected by the wf-core manifest versioned in this directory).

Not sure I follow. Without the version it will install the FP version defined by the manifest - so that will always be the 20.0.0.Beta2 and shouldn't change. Or am I missing something?

> BTW my goal was to have some inputs for immediate testing / demonstration, as there are no available manifests around yet and I always have to go and generate a fresh manifest from Wildfly. (I wanted to use wf-core because it's smaller / faster.)
👍 

> How does missing org.jboss.invocation:jboss-invocation-jakarta manifest? I don't see that being a wildfly-core dependency.

When I changed the core versions to 20.0.0.Beta2 provisioning failed with missing artifact. Maybe it was removed in Beta3